user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Tutorials/Incident_Tracking_Configuration_Tutorial.md

```diff
@@ -0,0 +1,313 @@
+
+<!-- > [!TIP]
+> See also: [Kata #43: [KATA] Grouping alarms like a pro](TODO: link and video reference still to be filled in here) -->
+
+## Prerequisites
+
+- DataMiner 10.2.0/10.1.4 or higher with [Storage as a Service (STaaS)](xref:STaaS) or a [self-hosted Cassandra database](xref:Supported_system_data_storage_architectures).
+
+- Automatic incident tracking is enabled in DataMiner Cube: *System Center* > *System settings* > *analytics config* > *Automatic incident tracking*.
+
+- Automatic incident tracking is enabled in the Alarm Console: To check if this is enabled, check if *Automatic incident tracking* is selected in the Alarm Console hamburger menu.
+
+  ![Enable Incident Tracking in the Alarm Console](~/user-guide/images/EnableAlarmGroupingInAlarmConsole.png)
+
+## Overview
+
+The tutorial consists of the following steps:
+
+- [Step 1: Install the example package from the Catalog](#step-1-install-the-example-package-from-the-catalog)
+- [Step 2: Find the advanced configuration file](#step-2-find-the-advanced-configuration-file)
+- [Step 3: Explore the default alarm grouping rules](#step-3-explore-the-default-alarm-grouping-rules)
+- [Step 4: Switch off parameter and view grouping](#step-4-switch-off-parameter-and-view-grouping)
+- [Step 5: Add your own rules to the incident tracking engine](#step-5-add-your-own-rules-to-the-incident-tracking-engine)
+- [Step 6: Group on a custom element property](#step-6-group-on-a-custom-element-property)
+- [Step 7: Clean up your system](#step-7-clean-up-your-system)
+
+## Step 1: Install the example package from the Catalog
+
+1. Go to <https://catalog.dataminer.services/details/5b989f43-bdab-4774-ae00-228b097363b1>.
+
+1. Deploy the Catalog item to your DataMiner Agent by clicking the *Deploy* button.
+
+   This will create 13 elements under the *DataMiner Catalog > Augmented Operations* > *Incident Tracking Tutorial* view. It will also create four Automation scripts. The names of all added elements and scripts starts with *Incident Tracking Tutorial*.
+
+   > [!NOTE]
+   > If the package has been fully deployed, but not everything mentioned above is visible in Cube, close and reopen Cube.
+
+## Step 2: Find the advanced configuration file
+
+Automatic incident tracking groups alarms according to a set of rules, which are listed in a configuration file. To locate the file in your system:
+
+1. Go to *Apps* > *System Center* > *System settings* > *analytics config*.
+
+1. Under *Automatic Incident Tracking*, check which ID is mentioned in the box *Leader DataMiner ID*.
+
+   This is the ID of the DataMiner Agent within your DMS that is in charge of running the incident tracking algorithm for the entire DMS.
+
+1. On the server running the leader DMA, go to the folder `C:\Skyline DataMiner\Analytics`.
+
+   Within this folder, you will find the *configuration.xml* file.
+
+1. Manually take a copy of your current *configuration.xml* file so that you can put it back when you finish this tutorial.
```

Comment:
Why should users take this manual copy? In step 7, it says a script will revert the configuration and the manual copy is not mentioned. Is this intended as a backup?

Reply:
The catalogue package contains 4 scripts: the last one being the reset script. Whenever you run script 1, 2 or 3 it creates a backup of your configuration.xml file if such a backup doesn't already exist. This backup is then used by script 4 to revert to the original setup. So, if you always use the automation scripts and don't do any manual changes to the configuration.xml, you don't need to worry about taking backups.

However, if you make any manual changes to the configuration.xml file before running any of the scripts, then the original configuration as you had it is gone (ie there is no backup of the original setup). Script 4 will then be unable to revert everything...

Reply:
I've updated the text to make sure this is clear. Could you take a look to check if everything is still OK for you?